docs/notebooks/bsync_examples/Small-Office-Level-2.md

```diff
@@ -0,0 +1,2179 @@
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("12:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("13:00:00", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(80.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("13:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("17:00:00", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(90.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("17:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("18:00:00", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(61.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("18:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("20:00:00", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(42.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("20:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("22:00:00", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(32.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("22:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("23:00:00", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(23.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("23:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("23:59:59", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(18.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Weekend"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("00:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("23:59:59", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(18.)
+    ),
+    bsync.ScheduleDetail(
+        bsync.DayType("Holiday"),
+        bsync.ScheduleCategory("Lighting"),
+        bsync.DayStartTime(datetime.strptime("00:00:00", "%H:%M:%S").time()),
+        bsync.DayEndTime(datetime.strptime("23:59:59", "%H:%M:%S").time()),
+        bsync.PartialOperationPercentage(18.)
+    )
+)
+schedule_light += bsync.LinkedPremises(
+    bsync.LinkedPremises.Building(
+        bsync.LinkedBuildingID(IDref=b1['ID'])
+    )
+)
+# plug load schedule
+schedule_pl += bsync.ScheduleDetails(
```

Comment:
this is great. I'd really like to have a helper function in `bsyncpy` to more rapidly generate these sections, something like 

def create_bsync_schedule_detail

where it takes in a tabular view of the data, then simple converts to the detail bsync. 

someday.

Reply:
Makes sense